netsim/templates/provider/clab/eos/ceos-config.j2

```diff
@@ -0,0 +1 @@
+SERIALNUMBER={{ ceos.serialnumber if ceos is defined and 'serialnumber' in ceos else 123456789 }}
```

Comment:
This would set the same serial number on all cEOS nodes in the lab. We learned the hard way with Cisco IOL that could be a bad idea. Just make the whole line depend on the existence of the serial number attribute

Reply:
Made it depend and insert a comment on how to configure it if desired. Also added systemmacaddr